tests/pytest_tests/system_tests/test_artifacts/test_wandb_artifacts.py

```diff
@@ -1475,3 +1476,43 @@ def test_cache_cleanup_allows_upload(wandb_init, tmp_path, monkeypatch):
     # Even though this works in production, the test often fails. I don't know why :(.
     assert found
     assert cache.cleanup(0) == 2**20
+
+
+@pytest.mark.parametrize(
+    "ttl_set,ttl_get,ttl_changed,ttl_is_inherited,ttl_duration_seconds",
+    [
+        ("DO_NOTHING", None, False, True, None),
+        (None, None, True, False, -2),
+        (ArtifactTTL.INHERIT, "ERROR", True, True, -1),
+        (
+            timedelta(days=100),
+            timedelta(days=100),
+            True,
+            False,
+            int(timedelta(days=100).total_seconds),
```

Comment:
How did you run this test? It does not work:
```
$ pytest tests/pytest_tests/system_tests/test_artifacts/test_wandb_artifacts.py -k test_artifact_ttl_setter_getter
...
tests/pytest_tests/system_tests/test_artifacts/test_wandb_artifacts.py:1492: in <module>
    int(timedelta(days=100).total_seconds),
E   TypeError: int() argument must be a string, a bytes-like object or a number, not 'builtin_function_or_method'
```

Reply:
fixed it